src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
-// Write code here
+import React from 'react';
+import './Person.scss';
+
+export function Person({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) {
+  let partnerSex = 'My husband`s name is ';
+
+  if (sex === 'm') {
+    partnerSex = 'My wife`s name is ';
+  }
+
+  return (
+    <>
+      <section className="Person">
+        <h2 className="Person__name">
+          My name is
+          {' '}
+          {name}
+        </h2>
+        <p className="Person__age">{age ? `I am ${age}` : null}</p>
```

Comment:
do you need a `p` with `null` value? I don't think so. So wrap your `p` tag with `age &&` condition